event/event.go

```diff
@@ -34,7 +34,7 @@ type Event struct {
 	CloudEventsVersion string                 `json:"cloudEventsVersion" validate:"required"`
 	Source             string                 `json:"source" validate:"uri,required"`
 	EventID            string                 `json:"eventID" validate:"required"`
-	EventTime          time.Time              `json:"eventTime,omitempty"`
+	EventTime          *time.Time             `json:"eventTime,omitempty"`
```

Comment:
I'm not sure I understand why we need this to be a pointer and not just the string representation of the `time.Now()`?